frontend/src/components/selectDevice/SelectDevice.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import QRCode from "react-qr-code";
+import styles from "./KeygenQRCode.module.css";
+interface KeygenQRCodeProps {
+  devices: string[];
+  selectedDevices: string[];
+  setSelectedDevices: React.Dispatch<React.SetStateAction<string[]>>;
+}
+const SelectDevice: React.FC<KeygenQRCodeProps> = ({
+  devices,
+  setSelectedDevices,
+}) => {
+  const { t } = useTranslation();
+
+  const checkHandler = (index: number) => {
+    const device = devices[index];
+    setSelectedDevices((prevSelectedDevices) =>
+      prevSelectedDevices.includes(device)
+        ? prevSelectedDevices.filter((d) => d !== device)
+        : [...prevSelectedDevices, device]
+    );
+  };
+
+  return (
+    <>
+      <h3 className="mt-5 font-semibold">
+        {t("keygen_peer_discovery_select_the_pairing_devices")}
+      </h3>
+      <div className="flex flex-wrap justify-center w-full items-center gap-10 my-2">
+        {devices.map((device, index) => (
+          <div
+            key={index}
+            className="relative bg-[#061B3A] rounded-lg text-center w-[120px] p-3 rounded-lg hover:border hover:border-white"
```

Comment:
check all hardcoded vaules if they are available in tailwindcss config
available => use it
not available and if this is used somewhere else like keysign etc, then define it and use it.

Reply:
Fixed